### PR TITLE
chore: fix linting setup

### PR DESCRIPTION
**Pull request by sparten11740**
Our linting setup was broken due to a mismatch in prettier versions used across the toolings. Linting also revealed a bunch of missing dev dependencies in `features/keychain`.

## Testplan
Linting passes locally

### CODE REVIEW

File: features/keychain/package.json [hunk 1]
```diff
@@ -45,6 +45,12 @@
     "p-defer": "^4.0.0"
   },
   "devDependencies": {
+    "@exodus/assets": "^9.1.1",
+    "@exodus/cardano-lib": "^1.4.4",
+    "@exodus/ethereum-lib": "^4.2.3",
+    "@exodus/solana-lib": "^1.7.5",
+    "@exodus/solana-meta": "^1.0.7",
+    "@exodus/storage-memory": "^2.1.1",
```
Comment (mvayngrib) [inline features/keychain/package.json:53]: we removed these for now because we're not ready to open source them yet #15

Comment (mvayngrib) [reply features/keychain/package.json:53]: those tests were separated out so they don't actually run. Maybe we can ignore lint errors there for now? Or i'm fine with removing them. We actually want to move them to the assets monorepo instead

Comment (sparten11740) [reply features/keychain/package.json]: disabled `import/no-unresolved` for tests for now: https://github.com/ExodusMovement/exodus-oss/pull/24/commits/aa076f419e64d7f595ecf9e325b016bf7814f3e1

**Review by mvayngrib — APPROVED** [review]

**Comment by mvayngrib** [comment]
thanks @sparten11740! i'll get this in tomorrow

**Pull request merged**